src/router/router.rs

```diff
@@ -196,62 +196,62 @@ fn creates_valid_regex_for_routes () {
 fn can_match_var_routes () {
     let route_store = &mut Router::<()>::new();
 
-    route_store.add_route(Method::Get, "/foo/:userid", middleware! { "hello from foo" });
-    route_store.add_route(Method::Get, "/bar", middleware! { "hello from foo" });
-    route_store.add_route(Method::Get, "/file/:format/:file", middleware! { "hello from foo" });
+    route_store.add_route(Method::GET, "/foo/:userid", middleware! { "hello from foo" });
+    route_store.add_route(Method::GET, "/bar", middleware! { "hello from foo" });
+    route_store.add_route(Method::GET, "/file/:format/:file", middleware! { "hello from foo" });
 
-    let route_result = route_store.match_route(&Method::Get, "/foo/4711").unwrap();
+    let route_result = route_store.match_route(&Method::GET, "/foo/4711").unwrap().0;
```

Comment:
This is unwrapping the `Ready(T)`, right?  Do we need to await it to avoid spurious failures?

Reply:
`route_match` isn't async since it doesn't do any i/o